src/graphics/device.js

```diff
@@ -899,6 +900,74 @@ pc.extend(pc, function () {
             }
         },
 
+        copyRenderTarget: function (source, dest, color, depth) {
+            var gl = this.gl;
+            var shaderCopy = false;
+
+            if (!this.webgl2 && depth) {
+                // #ifdef DEBUG
+                console.error("Depth is not copyable on WebGL 1.0");
+                // #endif
+                return false;
+            }
+            if (color) {
+                if (! (((source && source._colorBuffer) || !source) && ((dest && dest._colorBuffer) || !dest)) ) {
+                    // #ifdef DEBUG
+                    console.error("Can't copy color buffer, because one of the render targets doesn't have it");
+                    // #endif
+                    return false;
+                }
+            }
+            if (depth) {
+                if (! (((source && source._depthBuffer) || !source) && ((dest && dest._depthBuffer) || !dest)) ) {
+                    // #ifdef DEBUG
+                    console.error("Can't copy depth buffer, because one of the render targets doesn't have it");
+                    // #endif
+                    return false;
+                }
+            }
+            if (source) {
+                if (source && dest && source._colorBuffer && dest._colorBuffer && source._colorBuffer._format === dest._colorBuffer._format) {
+                    // #ifdef DEBUG
+                    console.error("Can't copy render targets of different formats");
+                    // #endif
+                    return false;
+                }
+            } else if (dest) {
+                if (source && dest && source._colorBuffer && dest._colorBuffer && source._colorBuffer._format === dest._colorBuffer._format) {
```

Comment:
Sorry, not sure how I missed that. Also, most of these checks aren't needed since we decided to not support copying from/to backbuffer, so source and dest must be always non-null now. Merged a fix.